samples/Location/audit-allowed-locations/README.md

```diff
@@ -0,0 +1,31 @@
+# Audit for approved datacentre locations
+
+Ensure approved datacentres are being utilised for deployed resources
+
+
+## Try on Portal
+
+
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/?feature.customportal=false&microsoft_azure_policy=true&microsoft_azure_policy_policyinsights=true&feature.microsoft_azure_security_policy=true&microsoft_azure_marketplace_policy=true#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-policy%2Fmaster%2Fsamples%2FLocation%2Faudit-allowed-locations%2Fazurepolicy.json)
+
+
+
+## Try with PowerShell
+
+````powershell
+$definition = New-AzureRmPolicyDefinition -Name "audit-location-deployments" -DisplayName "Audit for approved datacentre locations" -description "Ensure approved datacentres are being utilised for deployed resources" -Policy 'https://raw.githubusercontent.com/Azure/azure-policy/master/samples/Location/audit-allowed-locations/azurepolicy.rules.json' -Parameter 'https://raw.githubusercontent.com/Azure/azure-policy/master/samples/Location/audit-allowed-locations/azurepolicy.parameters.json' -Mode All
+$definition
+$assignment = New-AzureRMPolicyAssignment -Name <assignmentname> -Scope <scope>  -PolicyDefinition $definition
```

Comment:
the assignment here (and for CLI) needs to specify the listofAllowedLocations parameter